doc/examples/segmentation/plot_regionprops.py

```diff
@@ -22,9 +24,17 @@
 
 image = rotate(image, angle=15, order=0)
 
+rr, cc = ellipse(100, 100, 60, 50)
+image[rr, cc] = 1
```

Comment:
I have just realized that there is an issue with coordinates in ploting - https://scikit-image.org/docs/dev/auto_examples/segmentation/plot_regionprops.html. See how the axes are transposed with respect to the ellipse.

Reply:
Good catch! Fixing the major vs minor axes now.

Reply:
Done in cdf1678503c5dd55172fa322d3458727eb0758a6. Thanks again :)